File: src/main/java/confusedserver/goldeconomy/Events.java
package confusedserver.goldeconomy;

import org.bukkit.event.EventHandler;
import org.bukkit.event.player.PlayerJoinEvent;

import java.io.IOException;
import java.util.UUID;


public class Events implements org.bukkit.event.Listener {
   GoldEconomy goldEconomy;

    public Events(GoldEconomy goldEconomy) {
        this.goldEconomy = goldEconomy;
    }

    @EventHandler
   public void onPlayerJoining(PlayerJoinEvent e){
       String uuid = e.getPlayer().getUniqueId().toString();

       if (!goldEconomy.getBalanceFile().contains(uuid)){
           goldEconomy.getBalanceFile().set(uuid, 0);
       }
   }

}

File: src/main/kotlin/Events.kt
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerJoinEvent
import org.bukkit.event.player.PlayerQuitEvent

class Events(private var bank: Bank, private var util: Util) : Listener {

    @EventHandler
    fun onPlayerJoin(e: PlayerJoinEvent) {
        val uuid = e.player.uniqueId

//        util.createPlayer(uuid)

       if (!bank.playerAccounts.contains(uuid.toString())) {
           bank.playerAccounts[uuid.toString()] = 0
       }
    }

    @EventHandler
    fun onPlayerLeave(e: PlayerQuitEvent) {
        val uuid = e.player.uniqueId
        val balance = bank.getAccountBalance(uuid)

//        util.playerFileSet(uuid, "balance", balance)
    }
}